src/components/Popover/Popover.stories.tsx

```diff
@@ -147,3 +148,31 @@ export const LeftEnd: StoryObj<PopoverProps> = {
   },
   ...Default,
 };
+
+export const FocusClickableElement: StoryObj<PopoverProps> = {
+  play: async ({ canvasElement }) => {
+    // We want to test visual regression for the Popover.Content as well as the button,
+    // but don't want the drawer open initally outside Chromatic.
+    if (isChromatic()) {
+      const canvas = within(canvasElement);
+      const filtersTrigger = await canvas.findByRole('button');
+      userEvent.click(filtersTrigger);
```

Comment:
should this be `awaited`?

Reply:
It does look like `.click` returns a promise, so will prefix with `await` to be safe.